pkg/storage/bolt.go

```
@@ -30,7 +35,14 @@ func (b *BoltDB) Init(options interface{}) error {
 	if b.db != nil && b.IsOpen() {
 		return fmt.Errorf("bolit db already opened with name %s", b.URI())
 	}
-	dbFilePath := fmt.Sprintf("%s_%s.db", DBFilePrefix, b.Type())
+	_, err := os.Stat(DBRootPath)
+	if os.IsNotExist(err) {
+		err = os.MkdirAll(DBRootPath, os.ModePerm)
+		if err != nil {
+			panic(err)
```

let's return an error